app/javascript/app/layouts/sections/sections-styles.scss

```diff
@@ -1,13 +1,51 @@
 @import '~styles/settings.scss';
+@import '~styles/layout.scss';
 
 .page {
   min-height: calc(100vh - #{$header-height} - #{$footer-height});
 }
 
-.link {
-  color: $theme-color;
+.section {
+  background-image: linear-gradient(282deg, #007533, #005d28);
+  position: relative;
+
+  &::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-image: linear-gradient(282deg, #007533, #005d28);
+    z-index: -1;
+  }
+}
+
+.stickyWrapper {
+  max-height: 34px;
+
+  :global {
```

Comment:
I didn't know about that option! cool

Reply:
It is useful when you use external libraries to be able to access the classes from outside